packages/hub/routes/email-card-drop-requests.ts

```diff
@@ -74,44 +75,54 @@ export default class EmailCardDropRequestsRoute {
       return;
     }
 
+    ctx.type = 'application/vnd.api+json';
+
+    let claimedWithUserAddress = await this.emailCardDropRequestQueries.query({
+      ownerAddress: ctx.state.userAddress,
+      claimedAt: NOT_NULL,
```

Comment:
Why introduce a new utility, over using js to do the filtering?

`(await this.emailCardDropRequestQueries.query({ ownerAddress: ctx.state.userAddress, })).filter(r => r.claimedAt)`

Reply:
hmm well that does seem like it would work, I just think it’s pretty common to want to ask a database for `IS NOT NULL` and it’s sad to me that this query builder doesn’t permit it. Do you think it’s TOO ugly? Is there some better approach, maybe a `Symbol`?

Reply:
i think it's ok, was mainly asking because you describe this solution as hackish so i wondered if you would rather avoid it.